libs/design/src/atoms/form/quantity-field/quantity-input/quantity-input.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, Input, ViewChild, ChangeDetectionStrategy, ChangeDetectorRef, Optional, Self } from '@angular/core';
+import { ControlValueAccessor, NgControl } from '@angular/forms';
+import { DaffInputComponent } from '../../input/public_api';
+import { coerceNumberProperty } from '@angular/cdk/coercion';
+import { DaffFormFieldControl } from '../../form-field/public_api';
+
+@Component({
+  selector: 'daff-quantity-input',
+  templateUrl: './quantity-input.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [
+    {
+      provide: DaffFormFieldControl,
+      useExisting: DaffQuantityInputComponent
+    }
+  ]
+})
+export class DaffQuantityInputComponent implements ControlValueAccessor, DaffFormFieldControl {
+  @ViewChild(DaffInputComponent) input: DaffInputComponent;
+
+  /**
+   * @docs
+   * The minimum number for the quantity input field
+   */
+  @Input() min = 1;
+
+  /**
+   * @docs
+   * The maximum number for the quantity input field
+   */
+  @Input() max = 10;
+
+  focused = false;
+  disabled = false;
+  value = 1;
+  private regex: RegExp = new RegExp(/^[0-9]+$/g);
+  private specialKeys: Array<string> = ['Backspace', 'Tab', 'End', 'Home', 'ArrowRight', 'ArrowLeft', 'ArrowUp', 'ArrowDown'];
+  private onChange(quantity: number): void {};
+  private onTouched(obj: any): void {};
+
+  onInputKeyDown(event: KeyboardEvent) {
+    if (
+      this.specialKeys.indexOf(event.key) !== -1
+      || (
+        (event.key === 'a' || event.key === 'v' || event.key === 'c')
+        && (event.metaKey === true || event.ctrlKey === true)
+      )
+    ) {
+      return;
+    }
```

Comment:
Shouldn't `type="number"` force only number input text? At least add a comment here saying what this code block does bc its hard to read.